### PR TITLE
Fix various typos in comments and logging

### PR DESCRIPTION
**Pull request by github-john-doe**

### CODE REVIEW

File: lib/Runtime/Types/DynamicType.cpp [hunk 1]
```diff
@@ -85,7 +85,7 @@ namespace Js
     Currently we lock Both the Type and the Type Handler in the following cases:
     1. When a type is Shared.
     2. Snapshot Enumeration: DynamicType::PrepareForTypeSnapshotEnumeration()
-    To support snapshot enumetation we need to remember the type handler at the start of the enumeration in case the type evolves during enumeration. To acieve this we lock BOTH the
+    To support snapshot enumetation we need to remember the type handler at the start of the enumeration in case the type evolves during enumeration. To achieve this we lock BOTH the
```
Comment (dilijev) [inline lib/Runtime/Types/DynamicType.cpp]: Just noticed there's also: "enumetation" -> "enumeration"

**Comment by dilijev** [comment]
@dotnet-bot test Ubuntu shared_ubuntu_linux_release please

File: lib/Runtime/Types/DynamicType.cpp (hunk 1 above)
Comment (github-john-doe) [inline lib/Runtime/Types/DynamicType.cpp]: Fixed

**Comment by dilijev** [comment]
Squashed and merged in a2dd93ed8

**Comment by github-john-doe** [comment]
Thanks!

**Pull request closed without merging**